classes/class-gf-securesubmit.php

```diff
@@ -1342,24 +1336,15 @@ protected function buildCheckHolder($feed, $submission_data, $entry) {
      *
      * @return \HpsAddress
      */
-    private function buildAddress($feed, $entry) {
+    private function buildAddress($feed, $entry)
+    {
         $address = new HpsAddress();
-        if (in_array('billingInformation_address', $feed['meta'])) {
-            $address->address
-                = $entry[ $feed['meta']['billingInformation_address'] ] . $entry[ $feed['meta']['billingInformation_address2'] ];
-        }
-        if (in_array('billingInformation_city', $feed['meta'])) {
-            $address->city = $entry[ $feed['meta']['billingInformation_city'] ];
-        }
-        if (in_array('billingInformation_state', $feed['meta'])) {
-            $address->state = $entry[ $feed['meta']['billingInformation_state'] ];
-        }
-        if (in_array('billingInformation_zip', $feed['meta'])) {
-            $address->zip = $entry[ $feed['meta']['billingInformation_zip'] ];
-        }
-        if (in_array('billingInformation_country', $feed['meta'])) {
-            $address->country = $entry[ $feed['meta']['billingInformation_country'] ];
-        }
```

Comment:
@slogsdon Would anything other than the value hard coded cause an error? If not hard coded i would need to validate and apply the only valid entry if the value is valid? am I off base on that?